docs/user/experiments-api.md

```diff
@@ -39,7 +39,8 @@ assertEquals(
 
 ## Limits
 
-* Fixed maximum experiment and branch id lengths: 30. Longer strings are truncated. For the original Kotlin implementation of the Glean SDK, this is measured in Unicode characters. For the Rust implementation, this is measured in the number of bytes when the string is encoded in UTF-8.
+* `experimentId` and `branch` fields are fixed at a maximum length of 100. Fixed maximum key and value lengths for the `extra` map is 50. Longer strings used as ids, keys, or values are truncated to their respective maximum length. For the original Kotlin implementation of the Glean SDK, this is measured in Unicode characters. For the Rust implementation, this is measured in the number of bytes when the string is encoded in UTF-8.
```

Comment:
Let's not change current limits, please. This will introduce a different behaviour between glean-ac and glean-core. Let's limit this PR to the `extra` map.

Reply:
Again, was just trying to match what I found for desktop, but I can put it back for now.  Knowing what they wanted to name experiments and branches already (fairly long), I'm pretty sure we will need to address this soon.

Reply:
Let's keep this PR focused and address this problem when needed later on. Given we're going to migrate (hopefully) soon, we don't want to diverge implementation and create confusion in docs